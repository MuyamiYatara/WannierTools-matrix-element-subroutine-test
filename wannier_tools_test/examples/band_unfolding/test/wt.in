 &TB_FILE
 Hrfile = 'Graphene_hr.dat'
 /
 
 
 !> bulk band structure calculation flag
 &CONTROL
 BulkBand_Unfold_line_calc          = T
 BulkBand_Unfold_plane_calc         = T
 /
 
 &SYSTEM
 NSLAB = 10 
 NumOccupied =  1        ! NumOccupied
 SOC = 0                 ! soc
 E_FERMI =  -1.2533       ! e-fermi
 /
 
 &PARAMETERS
 Eta_Arc = 0.050     ! infinite small value, like brodening
 E_arc = -1.0         ! energy contour plot
 OmegaNum_unfold =600       ! omega number
 OmegaMin = -13.0     ! energy interval
 OmegaMax =  7.0     ! energy interval
 Nk1 =200           ! number k points
 Nk2 =200           ! number k points
 /
 

LATTICE
Angstrom
2.137711  -1.2342080   0.0000000        
0.0000000   2.4684160   0.0000000        
0.0000000   0.0000000   1.000000        

ATOM_POSITIONS
2                               ! number of atoms for projectors
Direct                          ! Direct or Cartisen coordinate
C 0.333333 0.666667 0.500000 C
C 0.666667 0.333333 0.500000 C

PROJECTORS
1 1        ! number of projectors
C  pz
C  pz

LATTICE_UNFOLD
Angstrom
2.137711  -1.2342080   0.0000000        
0.0000000   2.4684160   0.0000000        
0.0000000   0.0000000   1.000000    

ATOM_POSITIONS_UNFOLD
 2                               ! number of atoms for projectors
Direct                          ! Direct or Cartisen coordinate
C 0.333333 0.666667 0.500000 
C 0.666667 0.333333 0.500000 

PROJECTORS_UNFOLD
 1 1        ! number of projectors
C  pz
C  pz


SELECTED_ATOMS
1 
1-2


SURFACE
  0 0 1        
  1 0 0
  0 1 0
 
KPATH_BULK            ! k point path
4              ! number of k line only for bulk band

  M   -0.50000  1.00000  5.00000   K   -0.33333333  0.66666667  5.00000 
  K   -0.33333333  0.66666667  5.00000   G   0.00000  0.00000  5.00000 
  G   0.00000  0.00000 5.00000   K   0.33333333  -0.66666667  5.00000
  K   0.33333333  -0.66666667  5.00000  M   0.50000  -1.00000  5.00000

KPLANE_BULK
   0.00000   0.00000   5.00000
   3.00000   0.00000   0.00000
   0.00000   3.00000   0.00000
 
