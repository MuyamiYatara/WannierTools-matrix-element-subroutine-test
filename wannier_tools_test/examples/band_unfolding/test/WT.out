 -----------------------------------------------------------------------
  W            W            W              TTTTTTTTTTTTTTTTTTTT         
   W          W W          W                        TT                  
    W        W   W        W                         TT                  
     W      W     W      W                          TT                  
      W    W       W    W                           TT                  
       W  W         W  W                            TT                  
        WW           WW                             TT                  
        W             W                             TT                  
                                                                        
                        Welcome to WannierTools.                       
                             Version 2.6.1     
                   Tools for novel topological materials.               
                          Enjoy it and good luck.                       
                           Author : QuanSheng Wu                        
                        Email : wuquansheng@gmail.com                   
               Find more information on www.wanniertools.com            
 =======================================================================
                                                                        
 You are using   280 CPU cores
  
   
 >>>Read some paramters from wt.in
 You are using : TB     model
 Tight-binding Hamiltonian FROM: Hr File
 Is_Sparse_Hr=  F
 Tight-binding Hamiltonian filename: Graphene_hr.dat          
 System of particle: electron                 
 Tight-binding Hamiltonian obtained from : VASP                     
   
 >>>Control parameters: 
 BulkBand_line_calc                :  F
 BulkBand_plane_calc               :  F
 Landaulevel_unfold_line_calc      :  F
 BulkBand_unfold_line_calc         :  T
 BulkBand_unfold_plane_calc        :  T
 QPI_unfold_plane_calc             :  F
 BulkFatBand_calc                  :  F
 BulkBand_cube_calc                :  F
 BulkFS_calc                       :  F
 BulkFS_Plane_calc                 :  F
 BulkFS_plane_stack_calc           :  F
 BulkGap_cube_calc                 :  F
 BulkGap_plane_calc                :  F
 SlabBand_calc                     :  F
 SlabBandWaveFunc_calc             :  F
 SlabSS_calc                       :  F
 SlabArc_calc                      :  F
 SlabSpintexture_calc              :  F
 wanniercenter_calc                :  F
 BerryPhase_calc                   :  F
 BerryCurvature_calc               :  F
 BerryCurvature_EF_calc            :  F
 BerryCurvature_kpath_EF_calc      :  F
 BerryCurvature_kpath_Occupied_calc:  F
 BerryCurvature_Cube_calc          :  F
 BerryCurvature_slab_calc          :  F
 Dos_calc                          :  F
 Z2_3D_calc                        :  F
 WeylChirality_calc                :  F
 NLChirality_calc                  :  F
 Chern_3D_calc                     :  F
 MirrorChern_calc                  :  F
 JDos_calc                         :  F
 FindNodes_calc                    :  F
 EffectiveMass_calc                :  F
 AHC_calc                          :  F
 Boltz_OHE_calc                    :  F
 Boltz_Berry_correction            :  F
 LOTO_correction                   :  F
 OrbitalTexture_calc               :  F
 OrbitalTexture_3D_calc            :  F
 LandauLevel_k_calc                :  F
 LandauLevel_B_calc                :  F
 LandauLevel_wavefunction_calc     :  F
 Fit_kp_calc                       :  F
 DMFT_MAG_calc                     :  F
 Translate_to_WS_calc              :  F
 LandauLevel_kplane_calc           :  F
 LandauLevel_k_dos_calc            :  F
 LandauLevel_B_dos_calc            :  F
 FermiLevel_calc                   :  F
 Symmetry_Import_calc              :  F
 ChargeDensity_selected_bands_calc :  F
 ChargeDensity_selected_energies_calc :  F
   
 >>>System parameters: 
 NumSlabs :    10
 Nslab1 for ribbon  :     1
 Nslab2 for ribbon  :     1
 Number of Occupied bands:     1
 Number of total electrons:    2.000000
 With SOC or not in Hrfile:     0
 Fermi energy (eV) :       -1.253300
 surf_onsite (eV):         0.000000
 Add_Zeeman_Field:  F
 Zeeman_energy_in_eV (eV):         0.000000
 Electric_field_in_eVpA (eV/Angstrom):         0.000000
 Symmetrical_Electric_field_in_eVpA (eV/Angstrom):         0.000000
 Inner_symmetrical_Electric_Field:  F
 ijmax :    10
   
 **Notes**: There are two ways to specify magnetic field.
 1. specify Bmagnitude, Btheta, Bphi
 2. specify Bx, By, Bz
 Bmagnitude, Bx, By, Bz are real numbers in unit of Tesla. and Bx, By, Bz 
 are in the cartesian coordinates. Btheta is the angle with z axial 
  and Bphi is the angle with respect to x axial in the x-y plane
  Btheta is in [0, 180], Bphi is in [0, 360).
 If you specify both of them together, we will choose the first one.
 If choose the first one, but not specify Btheta, Bphi, then 
 by default we set Btheta=0, Bphi=0 which means B is along z direction.
  
  Warning: You didn't specify the magnitude of magnetic field.
  
 >>You did not specified the magnitude of magnetic field, 
  and also didn't set Bx, By, Bz. So we will set the magnetic direction
  from the settings of Btheta, Bphi.
  If you even didn't set Btheta, Bphi, we will take the default value, 
  Btheta=0, Bphi=0 which means B is along z direction
 >>You didn't set Btheta, and we set it to 0.
 >>You didn't set Bphi, and we set it to 0.
 Bmagnitude (in Tesla) :        1.000000
 Btheta, Bphi :        0.000000        0.000000
 Bx, By, Bz (in Tesla) :        0.000000        0.000000        1.000000
 B direction cosines :        0.000000        0.000000        1.000000
   
 >>>calculation parameters : 
 E_arc :         -1.00000
 Eta_arc :          0.05000
 Gap_threshold         0.01000
 OmegaMin :        -13.00000
 OmegaMax :          7.00000
 OmegaNum :    100
 OmegaNum_unfold :    600
 Nk1 :    200
 Nk2 :    200
 Nk3 :      1
 NP number of principle layers  :      2
 Tmin(Kelvin)  :        100.00000
 Tmax(Kelvin)  :        100.00000
 NumT  :      1
 NBTau  :      1
 Beta  :        100.00000
 Nslice_BTau_Max  :   5000
 BTauMax(Tesla.ps)         0.00000
 Relaxation_Time_Tau (ps)         1.00000
 Rcut    999999.00000
 Magp               1
 Magp_min               0
 Magp_max               1
 wcc_calc_tol         0.08000
 wcc_neighbour_tol         0.30000
 NumLCZVecs   400
 NumSelectedEigenVals     0
 NumRandomConfs:     1
 Projection weight mode:NORMAL              
 The size of magnetic supercell is Magq= :      10
  
 We found LATTICE card
>> lattice information (Angstrom)
           a           b           c       alpha        beta       gamma
    2.468416    2.468416    1.000000   47.625949   47.625949   63.501265
 Three Lattice vectors of the unfolded cell: 
    2.137711   -1.234208    0.000000
    0.000000    2.468416    0.000000
    0.000000    0.000000    1.000000
>> Reciprocal lattice information (1/Angstrom)
           a           b           c       alpha        beta       gamma
    2.939212    2.939212    6.283185  170.075351  170.075351  113.383568
 Three reciprocal lattice vectors of the primitive cell: 
    2.939212    0.000000    0.000000
    1.469606    2.545432    0.000000
    0.000000    0.000000    6.283185
  
 We found ATOM_POSITIONS card
Origin_cell%Num_atoms    2
  
 Warning: You didn't specify the atom magnetic moment in the ATOMIC_POSITION car
 d Or the format is wrong. So we set all the Atom-magnetic-moments to zero.
 
>> Atom position and magnetic moment of Original lattice
                                         Catesian                  Fractional(Direct)         Magnetic moment
------------------------------------------------------------------------------------------------------------------
 index  Atom Name            x           y           z           a           b           c      Mx      My      Mz
------------------------------------------------------------------------------------------------------------------
     1  C             1.346561    2.332317    0.944863    0.333333    0.666667    0.500000   0.000   0.000   0.000
     2  C             2.693127   -0.000002    0.944863    0.666667    0.333333    0.500000   0.000   0.000   0.000
  
 We found PROJECTORS card
 >>Number of projectors per atoms:
     1     1
C       pz      
C       pz      
  
 >> Wannier centers by default, in unit of unit lattice vectors
 iwann        R1        R2        R3         x         y         z
     1  0.333333  0.666667  0.500000  1.346561  2.332317  0.944863
     2  0.666667  0.333333  0.500000  2.693127 -0.000002  0.944863
  
 We found LATTICE_UNFOLD card
>> Folded lattice information (Angstrom)
           a           b           c       alpha        beta       gamma
    2.468416    2.468416    1.000000   47.625949   47.625949   63.501265
 Three Lattice vectors of unfolded cell: 
    2.137711   -1.234208    0.000000
    0.000000    2.468416    0.000000
    0.000000    0.000000    1.000000
>> Folded Reciprocal lattice information (1/Angstrom)
           a           b           c       alpha        beta       gamma
    2.939212    2.939212    6.283185  170.075351  170.075351  113.383568
 Three reciprocal lattice vectors of unfolded cell: 
    2.939212    0.000000    0.000000
    1.469606    2.545432    0.000000
    0.000000    0.000000    6.283185
 
>> The relation between the original lattice and the unfolded lattice
>> The cell volume ratio is:     1.000000
>> The lattice constant a ratio is:     1.000000
>> The lattice constant b ratio is:     1.000000
>> The lattice constant c ratio is:     1.000000
>> Origin_cell in unit of Folded_cell: R1      1.000000    0.000000    0.000000
>> Origin_cell in unit of Folded_cell: R2      0.000000    1.000000    0.000000
>> Origin_cell in unit of Folded_cell: R3      0.000000    0.000000    1.000000
 
  
 We found ATOM_POSITIONS_UNFOLD card
Folded_cell%Num_atoms    2
  
 Warning: You didn't specify the atom magnetic moment in the ATOMIC_POSITION car
 d Or the format is wrong. So we set all the Atom-magnetic-moments to zero.
 
C       0.333333    0.666667    0.500000
C       0.666667    0.333333    0.500000
 
>>> Atom position and magnetic moment of Unfolded lattice
                                    Catesian(Ang)                  Fractional(Direct)         Magnetic moment
------------------------------------------------------------------------------------------------------------------
 index  Atom Name            x           y           z           a           b           c      Mx      My      Mz
------------------------------------------------------------------------------------------------------------------
     1  C             0.712570    1.234209    0.500000    0.333333    0.666667    0.500000   0.000   0.000   0.000
     2  C             1.425141   -0.000001    0.500000    0.666667    0.333333    0.500000   0.000   0.000   0.000
  
 We found PROJECTORS_UNFOLD card
 >>Number of projectors per atoms:
     1     1
C       pz      
C       pz      
  
 We found SURFACE card
 
>> new vectors to define the surface (in unit of lattice vector)
 The 1st vector on surface     :    0.000000    0.000000    1.000000
 The 2nd vector on surface     :    1.000000    0.000000    0.000000
 The 3rd vector out of surface :    0.000000    1.000000    0.000000
  
 The rotated new unit cell in cartesian coordinates : 
    0.000000    0.000000    1.889726
    4.039688   -2.332315    0.000000
    0.000000    4.664630    0.000000
New cell's Volume is            5.27676Ang^3
  
 Fractional coordinates of atoms in units of new lattice vectors : 
C       0.500000    0.333333    0.666667
C       0.500000    0.666667    0.333333
  
 
 >> Top surface atom is      1 C         
 
 2D Primitive Cell_Volume:    2.46841603194944     
 Ra2, Rb2
    1.0000    0.0000
    0.0000    2.4684
 Ka2, Kb2
    1.7595    0.0000
    0.0000    0.7128
>> lattice information of the magnetic supercell (Angstrom)
    0.000000    0.000000    1.000000
    2.137711   -1.234208    0.000000
    0.000000   24.684160    0.000000
>> Reciprocal lattice information of the magnetic supercell (1/Angstrom)
    0.000000    0.000000    6.283185
    2.939212    0.000000    0.000000
    0.146961    0.254543    0.000000
  
3D Primitive Origin_cell%CellVolume:       5.27676004 in Angstrom^3
3D Magnetic supercell Volume:      52.76760036 in Angstrom^3
Projected area of magnetic supercell normal to the first vector specifed in SURFACE card:     188.43652159 in Angstrom^2
  
 We found KPATH_BULK card
Number of K lines :     4
 M     -0.5000   1.0000   5.0000  K      -0.3333   0.6667   5.0000
 K     -0.3333   0.6667   5.0000  G       0.0000   0.0000   5.0000
 G      0.0000   0.0000   5.0000  K       0.3333  -0.6667   5.0000
 K      0.3333  -0.6667   5.0000  M       0.5000  -1.0000   5.0000
 >> Kpoints plane for 2D system--> arcs  
K2D_start: -0.5000 -0.5000
The first vector:   1.0000  0.0000
The second vector:   0.0000  1.0000
  
 We found KPLANE_BULK card
 >> Kpoints plane for 3D system--> gapshape  
k3D_start :   0.0000  0.0000  5.0000
The 1st vector:   3.0000  0.0000  0.0000
The 2nd vector:   0.0000  3.0000  0.0000
 >> Kpoints cube for 3D system--> gapshape3D  
k3D_start :  0.0000  0.0000  0.0000
The 1st vector:   1.0000  0.0000  0.0000
The 2nd vector:   0.0000  1.0000  0.0000
The 3rd vector:   0.0000  0.0000  1.0000
kCubeVolume:  47.0081
ReciprocalOrigin_cell%CellVolume:  47.0081
  
  
 >> Using default parameters for Kpoint_3D
>> k points  0.0000 0.0000 0.0000 in unit of reciprocal primitive cell
>> k points  0.0000 0.0000 0.0000 in Cartesian coordinates
  
 >> Using default parameters for effective mass calculation
 >> Effective mass calculation parameters  
>> The     1'th band
>> k step  0.0100 in unit of 1/Angstrom
>> k points  0.0000 0.0000 0.0000 in unit of reciprocal primitive cell
>> k points  0.0000 0.0000 0.0000 in unit of 1/Angstrom
  
 >> We use the default values for k3points_pointmode_direct=[0,0,0]
 >> and Nk3_point_mode = 1
  
 >> We use the default values for k3points_unfold_pointmode_direct=[0,0,0]
 >> and Nk3_unfold_point_mode = 1
  
 >> We use the default values for Single_KPOINT_2D_DIRECT=[0,0]
  
 >> We use the default values for Single_KPOINT_3D_DIRECT=[0,0,0]
 >> Output all atoms weight for surface state spectrum
 > NtopAtoms            2
> TopAtoms 
     1     2
> NbottomAtoms          2
> BottomAtoms 
     1     2
 > NtopOrbitals            2
> TopOrbitals 
     1     2
> NBottomOrbitals    2
> BottomOrbitals 
     1     2
  
 >> We do not calculate chirality for nodal lines
  
 >> We do not calculate chirality for weyl points
  
 We found SELECTED_ATOMS card
  
>> There are    1 groups of SelectedAtoms
Group :   1
>> Number of atoms selected         2
>> Selected atoms are
    1  C             2  C         
  
>> There are   1 groups of SelectedOrbitals
 >> SelectedOrbitals
>> Number of orbitals selected (including spin degenarcy)         2
>> Orbitals are
       1       2
  
 >> SELECTED_OCCUPIED_BANDS
>> Number of Occupied bands selected          1
>> OccupiedBand indices are
     1
  
  
 >> SELECTEDBANDS
>> Number of bands selected          2
>> Band indices are
     1     2
  
 <<<Read wt.in file successfully
 
  File Graphene_hr.dat exist, We are using HmnR from wannier90
 >> Num_wann           2
 >> NRPTS         315
  >> Begin to read Hmn_R.data
  
 
  Note: This is not a 2D system or E=0, so we can't add electric field.
 
  << Read Hmn_R.data successfully
  
 >> We finished the symmetry reducing of the kpoints in the kcube
  Number of kpoints mesh is      40000
  Number of symmetry reduced kpoints is          0
  
  
 >> Start of unfolding bulk band
BulkBand unfolding at :         0         1 /       800
 n           1
 n           2
BulkBand unfolding at :         0       281 /       800
 n           1
 n           2
BulkBand unfolding at :         0       561 /       800
 n           1
 n           2
 loop done
 <<< Unfold bands successfully
 Time cost for BulkBand_unfold_line_calc is about           10.902 s
 << End of unfolding bulk band
  
 >> Start of unfolding bulk band in plane mode
BulkBand unfolding at :         1 /*
BulkBand unfolding at :       281 /*
BulkBand unfolding at :       561 /*
BulkBand unfolding at :       841 /*
BulkBand unfolding at :      1121 /*
BulkBand unfolding at :      1401 /*
BulkBand unfolding at :      1681 /*
BulkBand unfolding at :      1961 /*
BulkBand unfolding at :      2241 /*
BulkBand unfolding at :      2521 /*
BulkBand unfolding at :      2801 /*
BulkBand unfolding at :      3081 /*
BulkBand unfolding at :      3361 /*
BulkBand unfolding at :      3641 /*
BulkBand unfolding at :      3921 /*
BulkBand unfolding at :      4201 /*
BulkBand unfolding at :      4481 /*
BulkBand unfolding at :      4761 /*
BulkBand unfolding at :      5041 /*
BulkBand unfolding at :      5321 /*
BulkBand unfolding at :      5601 /*
BulkBand unfolding at :      5881 /*
BulkBand unfolding at :      6161 /*
BulkBand unfolding at :      6441 /*
BulkBand unfolding at :      6721 /*
BulkBand unfolding at :      7001 /*
BulkBand unfolding at :      7281 /*
BulkBand unfolding at :      7561 /*
BulkBand unfolding at :      7841 /*
BulkBand unfolding at :      8121 /*
BulkBand unfolding at :      8401 /*
BulkBand unfolding at :      8681 /*
BulkBand unfolding at :      8961 /*
BulkBand unfolding at :      9241 /*
BulkBand unfolding at :      9521 /*
BulkBand unfolding at :      9801 /*
BulkBand unfolding at :     10081 /*
BulkBand unfolding at :     10361 /*
BulkBand unfolding at :     10641 /*
BulkBand unfolding at :     10921 /*
BulkBand unfolding at :     11201 /*
BulkBand unfolding at :     11481 /*
BulkBand unfolding at :     11761 /*
BulkBand unfolding at :     12041 /*
BulkBand unfolding at :     12321 /*
BulkBand unfolding at :     12601 /*
BulkBand unfolding at :     12881 /*
BulkBand unfolding at :     13161 /*
BulkBand unfolding at :     13441 /*
BulkBand unfolding at :     13721 /*
BulkBand unfolding at :     14001 /*
BulkBand unfolding at :     14281 /*
BulkBand unfolding at :     14561 /*
BulkBand unfolding at :     14841 /*
BulkBand unfolding at :     15121 /*
BulkBand unfolding at :     15401 /*
BulkBand unfolding at :     15681 /*
BulkBand unfolding at :     15961 /*
BulkBand unfolding at :     16241 /*
BulkBand unfolding at :     16521 /*
BulkBand unfolding at :     16801 /*
BulkBand unfolding at :     17081 /*
BulkBand unfolding at :     17361 /*
BulkBand unfolding at :     17641 /*
BulkBand unfolding at :     17921 /*
BulkBand unfolding at :     18201 /*
BulkBand unfolding at :     18481 /*
BulkBand unfolding at :     18761 /*
BulkBand unfolding at :     19041 /*
BulkBand unfolding at :     19321 /*
BulkBand unfolding at :     19601 /*
BulkBand unfolding at :     19881 /*
BulkBand unfolding at :     20161 /*
BulkBand unfolding at :     20441 /*
BulkBand unfolding at :     20721 /*
BulkBand unfolding at :     21001 /*
BulkBand unfolding at :     21281 /*
BulkBand unfolding at :     21561 /*
BulkBand unfolding at :     21841 /*
BulkBand unfolding at :     22121 /*
BulkBand unfolding at :     22401 /*
BulkBand unfolding at :     22681 /*
BulkBand unfolding at :     22961 /*
BulkBand unfolding at :     23241 /*
BulkBand unfolding at :     23521 /*
BulkBand unfolding at :     23801 /*
BulkBand unfolding at :     24081 /*
BulkBand unfolding at :     24361 /*
BulkBand unfolding at :     24641 /*
BulkBand unfolding at :     24921 /*
BulkBand unfolding at :     25201 /*
BulkBand unfolding at :     25481 /*
BulkBand unfolding at :     25761 /*
BulkBand unfolding at :     26041 /*
BulkBand unfolding at :     26321 /*
BulkBand unfolding at :     26601 /*
BulkBand unfolding at :     26881 /*
BulkBand unfolding at :     27161 /*
BulkBand unfolding at :     27441 /*
BulkBand unfolding at :     27721 /*
BulkBand unfolding at :     28001 /*
BulkBand unfolding at :     28281 /*
BulkBand unfolding at :     28561 /*
BulkBand unfolding at :     28841 /*
BulkBand unfolding at :     29121 /*
BulkBand unfolding at :     29401 /*
BulkBand unfolding at :     29681 /*
BulkBand unfolding at :     29961 /*
BulkBand unfolding at :     30241 /*
BulkBand unfolding at :     30521 /*
BulkBand unfolding at :     30801 /*
BulkBand unfolding at :     31081 /*
BulkBand unfolding at :     31361 /*
BulkBand unfolding at :     31641 /*
BulkBand unfolding at :     31921 /*
BulkBand unfolding at :     32201 /*
BulkBand unfolding at :     32481 /*
BulkBand unfolding at :     32761 /*
BulkBand unfolding at :     33041 /*
BulkBand unfolding at :     33321 /*
BulkBand unfolding at :     33601 /*
BulkBand unfolding at :     33881 /*
BulkBand unfolding at :     34161 /*
BulkBand unfolding at :     34441 /*
BulkBand unfolding at :     34721 /*
BulkBand unfolding at :     35001 /*
BulkBand unfolding at :     35281 /*
BulkBand unfolding at :     35561 /*
BulkBand unfolding at :     35841 /*
BulkBand unfolding at :     36121 /*
BulkBand unfolding at :     36401 /*
BulkBand unfolding at :     36681 /*
BulkBand unfolding at :     36961 /*
BulkBand unfolding at :     37241 /*
BulkBand unfolding at :     37521 /*
BulkBand unfolding at :     37801 /*
BulkBand unfolding at :     38081 /*
BulkBand unfolding at :     38361 /*
BulkBand unfolding at :     38641 /*
BulkBand unfolding at :     38921 /*
BulkBand unfolding at :     39201 /*
BulkBand unfolding at :     39481 /*
BulkBand unfolding at :     39761 /*
BulkBand unfolding at :     40041 /*
BulkBand unfolding at :     40321 /*
 <<< Unfold bands successfully
 Time cost for BulkBand_unfold_plane_calc is about            9.416 s
 << End of unfolding bulk band in plane mode
  
 Time cost for whole program is about           26.193 s
  
  
  =======================================================================
  Congratulations! you finished the calculation.
  I hope you could find something useful from this calculation.
  If so, I would like to ask you to cite our this paper:
  
  WannierTools : An open-source software package for novel topological materials
  QuanSheng Wu and ShengNan Zhang and Hai-Feng Song and Matthias Troyer and Alexey A. Soluyanov
  Computer Physics Communications 224, 405 (2018)
  https://doi.org/10.1016/j.cpc.2017.09.033
  
  For bugs, please report to wuquansheng@gmail.com                   
  or wanniertools@groups.google.com.                 
  More information could find on www.wanniertools.com            
  See you next time :)
  =======================================================================
